docs/examples/source_files/qaoa/qaoa.py

```diff
@@ -66,11 +66,11 @@ def run_qaoa(
         # if we have service we need to open a session and create sampler
         service = arguments.get("service")
         backend = arguments.get("backend", "ibmq_qasm_simulator")
-        with Session(service=service, backend=backend) as session:
-            options = Options()
-            options.optimization_level = 3
+        session = Session(service=service, backend=backend)
+        options = Options()
+        options.optimization_level = 3
 
-            estimator = Estimator(session=session, options=options)
+        estimator = Estimator(session=session, options=options)
```

Comment:
I can't comment earlier in this file, but when trying to run the QAOA notebook I had to delete the import of `QuantumServerless` in https://github.com/akihikokuroda/quantum-serverless/blob/sidecarproxy/docs/examples/source_files/qaoa/qaoa.py#L14

Reply:
fixed while I was writing my review :smile: